package.json

```diff
@@ -62,7 +62,7 @@
     "i18next-http-backend": "^2.0.0",
     "livekit-client": "^2.0.2",
     "lodash": "^4.17.21",
-    "matrix-js-sdk": "github:matrix-org/matrix-js-sdk#d55c6a36df539f6adacc335efe5b9be27c9cee4a",
+    "matrix-js-sdk": "github:AndrewFerr/matrix-js-sdk#msc-3266-compliance",
```

Comment:
This can be reverted back to upstream, either on `#develop` or `#e874468ba3e84819cf4b342d2e66af67ab4cf804` (which is commit for matrix-org/matrix-js-sdk#4155).